### PR TITLE
fix chapter list not reflecting blocked groups from mangadex

### PR DESCRIPTION
**Pull request by CarlosEsco**
Force network instead of using cache when getting manga info and chapter list for mangadex.

### CODE REVIEW

File: src/all/mangadex/src/eu/kanade/tachiyomi/extension/all/mangadex/Mangadex.kt [hunk 1]
```diff
@@ -347,13 +347,19 @@ abstract class Mangadex(
     }
 
     private fun apiRequest(manga: SManga): Request {
-        return GET(baseUrl + API_MANGA + getMangaId(manga.url), headers)
```
Comment (happywillow0) [inline src/all/mangadex/src/eu/kanade/tachiyomi/extension/all/mangadex/Mangadex.kt]: Doesn't the GET request have a third comma separated value for cache control?
GET(`url` , `headers`, `cache control`) ?

**Comment by CarlosEsco** [comment]
Oh true

**Comment by joseph619** [comment]
Would this also prevent chapters from blocked groups being automatically downloaded? Preferably they would be blocked from downloads if Autodownload new chapters is enabled.

**Comment by happywillow0** [comment]
It should in theory. It all depends on what Mangadex is giving us. 
If they do not show up the Mangadex api / site, then they should not get downloaded. 

The only thing I can think of is what happens if a user gets logged out without them knowing and the chapter list gets refreshed. _Then_ the blocked group may re-appear and will get downloaded.

**Comment by CarlosEsco** [comment]
Yes. This does prevent their API from sending it back.  It should currently do that also though. This just bypasses the sometimes cached call

File: src/all/mangadex/src/eu/kanade/tachiyomi/extension/all/mangadex/Mangadex.kt (hunk 1 above)
Comment (CarlosEsco) [reply src/all/mangadex/src/eu/kanade/tachiyomi/extension/all/mangadex/Mangadex.kt]: fixed.  This is why i waited for a review :)  knew i was probably overlooking something simple

**Pull request merged**